proto/tendermint/state/types.proto

```diff
@@ -41,7 +41,8 @@ message State {
   Version version = 1 [(gogoproto.nullable) = false];
 
   // immutable
-  string chain_id = 2 [(gogoproto.customname) = "ChainID"];
+  string chain_id       = 2 [(gogoproto.customname) = "ChainID"];
+  int64  initial_height = 14;
```

Comment:
is this aimed at grouping releative fields?

Reply:
Yes, these are all immutable.